src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+                        https://docs.google.com/presentation/d/1qvGtOdH6<br/>_qirfyPLq628x1t8YId_kYMFndaaecIpHSA/edit?usp=sharing</Link><br/>
+                    </Text>
+                </ColumnText>
+            </Columns>       
+        </div>
+    )
+}
+
+const ThirdPlace = () => {
+    return (
+        <div>
+            <Subtitle css={`margin-top: 2vw`}>Third Place - Team Chaeto</Subtitle>
+
+            <Columns css={`text-align: center`}>
+                <img src={TeamChaeto} alt="Team Chaeto" css={`height: 25vw`}/>
+                <figure>
+                    <img src={ChrisNick} alt="Christopher and Nicholas Kwok" css={`height: 25vw`}/>
+                    <Name>Christopher and Nicholas Kwok</Name>
+                </figure>
+            </Columns>
+
+            
+            <Text>Team Chaeto pioneers an ecological, sustainable, and feasible solution 
+                to the existential crisis of ocean acidification. By modifying the activity 
+                of hydrogenase and carbonic anhydrase enzymes in chaetomorpha, the team 
+                invented chaeto biofilters to implement their novel strain of algae in 
+                marine ecosystems.</Text>
+
+            <Columns>
+                <ColumnText css={`padding-right: 2vw`}>
+                    <b><Subtitle fontSize="0.6rem">What was competing in the PCV Prize like?</Subtitle></b>
+                    <Text>"Competing in the Penn Climate Venture Prize was an eye-opening 
+                        experience! Meeting the collegiate and high school teams with their 
+                        startups was inspirational. As high school students, the competition 
+                        was an opportunity to navigate entrepreneurship with incredible mentors."<br/>
+                        <br/>
+                        <b>Find Team Chaeto on Instagram:</b> <br/>
+                        <Link href="https://www.instagram.com/chris_kwok04/">@chris_kwok04</Link> <br/>
+                        <Link href="https://www.instagram.com/nicholas_kwok04/">@nicholas_kwok04</Link>
+                        </Text>
+                </ColumnText>
+                <ColumnText css={`padding-left: 2vw`}>
+                    <b><Subtitle fontSize="0.6rem">Future Plans?</Subtitle></b>
+                    <Text>"In the near future, Team Chaeto plans to continue research and 
+                        development with their chaeto biofilters as an economically viable 
+                        solution to ocean acidification. Furthermore, the company seeks 
+                        numerous grants to raise capital and scale production. Eventually, 
+                        Team Chaeto will pursue government contracts and partnerships with 
+                        environmental agencies to market their solution globally."<br/>
+                    </Text>
+                </ColumnText>
+            </Columns>   
+        </div>
+    )
+}
+
+const HonorableMention = () => {
```

Comment:
These components are basically the same so critiques on each probably apply to all of them FYI